tests/basic/storage_pool_test.go

```diff
@@ -6880,7 +6880,9 @@ var _ = Describe("{DriveAddPXDown}", func() {
 
 		// Add Drive on the Node [ PTX-15856 ]
 		err = AddCloudDrive(*nodeDetail, -1)
-		log.FailOnError(err, "adding new pool on the node failed?")
+		poolExpandCondition := strings.Contains(err.Error(), "add cloud drive failed on node")
+		dash.VerifyFatal(poolExpandCondition, true, "adding new pool on the node failed?")
+		log.Errorf(err.Error(), "Pool expand failed")
```

Comment:
do not log as error when error is expected

Reply:
Done Please check